test/angular-toggle-switch.spec.js

```diff
@@ -22,6 +25,7 @@ describe('Toggle Switch', function() {
     var elm = angular.element(template);
     $compile(elm)(scope);
     scope.$apply();
+    isolateScope = elm.isolateScope();
```

Comment:
to keep the `expect()` simple in the tests -- since jQuery isn't available, I wasn't sure how exactly to do the equivalent of e.g., `elm.find('.knob-label')`. But might as well test what the directive sets within its isolate scope anyway, since that doesn't rely on the css class.